### ISSUE TITLE
JUnit results not recognizing errors when there are multiple test suites

### ISSUE THREAD

**Issue by jacobdfriedmann** [bugcrowd/test-summary-buildkite-plugin#9]
Jest's JUnit plugin outputs (I think valid) XML that has multiple test suites:

```xml
<testsuites name="jest tests" tests="2" failures="1" time="7.298">
<testsuite name="Login" errors="0" failures="1" skipped="0" timestamp="2018-05-08T23:03:15" time="7.298" tests="2">
<testcase classname="Login should load without error" name="Login should load without error" time="2.735">
<failure>
Error: Expected image to match or be a close match to snapshot but was 8.546910755148742% different from snapshot (6723 differing pixels). See diff for details: /app/__tests__/__image_snapshots__/__diff_output__/login-js-login-should-load-without-error-2-diff.png at Object.<anonymous>.module.exports (/app/src/screenshot.js:41:17) at <anonymous> at process._tickCallback (internal/process/next_tick.js:188:7)
</failure>
</testcase>
<testcase classname="Login should log in successfully" name="Login should log in successfully" time="0.028"></testcase>
</testsuite>
</testsuites>
```

However, the plugin doesn't seem to recognize the failure:

```
Status: Image is up to date for tessereth/test-summary-buildkite-plugin:v1.2.0
--
  | $ buildkite-agent artifact download reports/e2e-junit.xml tmp/test-summary
  | 2018-05-08 23:04:10 INFO   Searching for artifacts: "reports/e2e-junit.xml"
  | 2018-05-08 23:04:11 INFO   Found 1 artifacts. Starting to download to: /usr/src/app/tmp/test-summary
  | 2018-05-08 23:04:11 INFO   Successfully downloaded "reports/e2e-junit.xml" 928 bytes
  |  
  | No errors found! 🎉


```

**Comment by tessereth** [comment]
Ah, interesting. Thanks for reporting it! I'll have a look as soon as I get a chance.

**Comment by tessereth** [comment]
So, yes `testsuites` are valid and they're easy enough to handle. Thanks for bringing it up and it should be fixed in the next release. But on an unrelated note, your error message includes unescaped `<anonymous>` strings which means the xml parser fails to parse the document. Maybe that's a bug for Jest? I think they should either XML escape the failure string or wrap it in `<![CDATA[...]]>`.

**Comment by tessereth** [comment]
The multiple testsuites fix is included in `v1.3.0`. Let me know how it goes.
